charts/nimbus/values.yaml

```diff
@@ -274,10 +284,30 @@ updateStrategy:
   type: RollingUpdate
 
 # -- Additional init containers
-initContainers: []
-# - name: my-init-container
-#   image: busybox:latest
-#   command: ['sh', '-c', 'echo hello']
+initContainers:
```

Comment:
Can you add this and rerun make docs?

```
# @default -- See `values.yaml`
```

So that the readme doesn't include the whole init container example.